docs/arista.eos.eos_logging_module.rst

```diff
@@ -16,7 +16,7 @@ Version added: 1.0.0
 
 DEPRECATED
 ----------
-:Removed in collection release after 2024-01-01
+:Removed in collection release after 2023-09-01
```

Comment:
It looks like the documentation was changed here without commiting the module update?